src/utils/gauges.ts

```diff
@@ -130,14 +133,17 @@ export async function getGaugeData(
         registryAddresses,
         account,
       )
-    } else {
+    } else if (shouldLoadChildGauges(chainId)) {
       return buildGaugeDataSidechain(
         library,
         chainId,
         basicPools,
         registryAddresses,
         account,
       )
+    } else {
+      console.error(`Gauges are not supported on chainId: ${chainId}`)
```

Comment:
I think this is better.  `console.error("Gauges are not supported on ChainId[chainId].")`